niri-config/src/lib.rs

```diff
@@ -2339,6 +2339,8 @@ pub struct DebugConfig {
     #[knuffel(child)]
     pub disable_direct_scanout: bool,
     #[knuffel(child)]
+    pub disable_set_bpc: bool,
```

Comment:
Maybe `keep_max_bpc_unchanged`?

Also needs to be mentioned in the debug options wiki page.

Reply:
Added and renamed